templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
```

Comment:
Hmm -- interesting!  That is worth a try, I guess.  I wasn't aware that the cloud provider was that smart.